pkg/sync/rpc/server.go

```diff
@@ -682,17 +682,23 @@ func (s *SyncAgentServer) BackupCreate(ctx context.Context, req *ptypes.BackupCr
 		Labels:               req.Labels,
 	})
 	if err != nil {
-		logrus.WithError(err).Errorf("Failed to create backup %v", req.BackupName)
+		logrus.WithError(err).Errorf("Error initializing backup %v", req.BackupName)
```

Comment:
Suggest log `Failed to initialize backup %v`

Reply:
Updated. Thank you.